resources/aws/errors.go

```diff
@@ -13,6 +13,7 @@ var (
 
 	gatewayFindError       = errgo.New("Couldn't find gateway")
 	routeTableFindError    = errgo.New("Couldn't find route table")
+	routeFindError         = errgo.New("Couldn't find route")
```

Comment:
Awesome. I added these points to https://github.com/giantswarm/aws-operator/issues/112#issuecomment-295644927, as a form of documentation. Not sure if we could enforce this constraint (and others) automatically somehow. Maybe there's a tool for that we could run in the CI?